### DISCUSSION TITLE
How to specify a slot of range `float` that accepts values less than `10.0`?

### DISCUSSION THREAD

**Discussion by candleindark** [linkml/linkml#2144] [Q&A]
How to specify a slot of range `float` that accepts values less than `10.0`?

I am aware of the meta slot [`maximum_value`](https://linkml.io/linkml-model/latest/docs/maximum_value/). However, according to the documentation, the value for that meta slot seems to constraint only ordinal ranges such as integer, and it's considered to be an acceptable value for the slot being defined. I am looking for something that is comparable to `exclusiveMaximum` in JSON schema.

Ping @djarecka

**Comment by cmungall** [comment]
Hi @candleindark!

You can in fact use floats with maximum and minimum, but there is not yet an equivalent of exclusivemin/max, it's currently inclusive only - this will be added in the next metamodel.

You can make a custom rule but this is less satsifactory..

**Reply by candleindark** [reply]
Thanks for the clarification. Do do you know when the a next metamodel will be available?

Would you illustrate how to accomplish this using a custom rule?
